src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -129,15 +130,6 @@ public void setPrivacy(Privacy privacy) throws IOException {
         root.createRequest().method("PATCH").with("privacy", privacy).withUrlPath(api("")).send();
     }
 
-    /**
-     * Gets id.
-     *
-     * @return the id
-     */
-    public int getId() {
```

Comment:
We have a `BridgeMethods` to handle cases like these.  Preserving backward compatibility while changing method return types. 

https://github.com/github-api/github-api/blob/7d842175f7984e33b3d97346cba3b20a1f905031/src/main/java/org/kohsuke/github/GHObject.java#L75

Reply:
Seems like it works only at runtime..
IDE still shows errors when trying to use it to return `int`..

But i assume its ok..

Reply:
I'm not sure this is okay.  If I had: 
```
int a = team.getId();
```

Does it compile still with this change?

Reply:
` incompatible types: possible lossy conversion from long to int`